### PR TITLE
refactored createKeyStream && createValueStream

### PR DESCRIPTION
**Pull request by alessioalex**
tl;dr 'method too smart err'

The reason for this is simple: I thought this module would be used only on top of `level`, so when you call `createKeyStream` that would call the original function from `level` in the context of the `level-prefix` instance. However if there are other modules such as this one that wrap the `level` instance and use some special property / function from the prototype in the `createKeyStream` / `createValueStream` that would break, since `this` would be referring to `level-prefix`.

If you're still unsure on what I'm saying there's a more concrete example: try to use `level-prefix` on top of a `level-store` which uses `multilevel-http` client:

https://gist.github.com/alessioalex/7341121

Can you please push another version on NPM after you merge this? Thanks!

### CODE REVIEW

**Comment by juliangruber** [comment]
i knew it :D

**Comment by juliangruber** [comment]
published as 0.2.0
